onnxruntime/contrib_ops/webgpu/quantization/subgroup_matrix_matmul_nbits.cc

```diff
@@ -69,7 +71,39 @@ Status SubgroupMatrixMatMulNBitsProgram::GenerateShaderCode(ShaderHelper& shader
                 tile_B[tile_b_base + 7] = b_value_upper[3];
             }
         }
-
+    )ADDNL_FN";
+  } else {
```

Comment:
nit: if (nbits_ == 8) {

this way the program will fail correctly for nbits other than 4/8.